app/assets/javascripts/lib/ui_helper.js

```diff
@@ -0,0 +1,69 @@
+Skull.UI_helper = {
+
+	setupResponsiveIframes: function($container){
+	    
+		this.$container = $container;
+
```

Comment:
Too much empty lines, and empty line should be used to separate related lines, if we use it everywhere it just makes the code sparser.